next/pages/guidelines/color/palette.tsx

```diff
@@ -0,0 +1,406 @@
+import React, { useState } from 'react';
+import type { InferGetStaticPropsType } from 'next';
+import ClickableBox from 'clickable-box';
+import { groupBy } from 'lodash-es';
+import classNames from 'classnames';
+import * as tokens from '@thumbtack/thumbprint-tokens';
+import { NavigationCaretDownSmall, NavigationCaretUpSmall } from '@thumbtack/thumbprint-icons';
+import { Text } from '@thumbtack/thumbprint-react';
+import { ContentPage } from '../../../components/mdx/mdx';
+import getLayoutProps from '../../../utils/get-layout-props';
+import { H2, P } from '../../../components/mdx/components';
+import { paletteColortMappings } from './usage-mappings';
+import { Color, Image, Usage } from './utils';
+
+import purple from '../../../images/pages/guide/product/color/palette/purple.png';
+import yellow from '../../../images/pages/guide/product/color/palette/yellow.png';
+import neutral from '../../../images/pages/guide/product/color/palette/neutral.png';
+import red from '../../../images/pages/guide/product/color/palette/red.png';
+import blue from '../../../images/pages/guide/product/color/palette/blue.png';
+import green from '../../../images/pages/guide/product/color/palette/green.png';
+import indigo from '../../../images/pages/guide/product/color/palette/indigo.png';
+
+const images: Image = {
+    purple: {
+        src: purple,
+        alt: 'user interface example where purple colors are applied',
+    },
+    yellow: {
+        src: yellow,
+        alt: 'user interface example where yellow colors are applied',
+    },
+    neutral: {
+        src: neutral,
+        alt: 'user interface example where neutral colors are applied',
+    },
+    red: {
+        src: red,
+        alt: 'user interface example where red colors are applied',
+    },
+    blue: {
+        src: blue,
+        alt: 'user interface example where blue colors are applied',
+    },
+    green: {
+        src: green,
+        alt: 'user interface example where green colors are applied',
+    },
+    indigo: {
+        src: indigo,
+        alt: 'user interface example where indigo colors are applied',
+    },
+};
+
+interface ColoredPillProps {
+    fill: string;
+    title: string;
+    value: string;
+}
+
+function TokenPill({ fill, title, value }: ColoredPillProps): JSX.Element {
+    return (
+        <div>
+            <div>{title}</div>
+            <div
+                className="ph3 pv2 br-pill"
+                style={{
+                    background: `${fill}`,
+                }}
+            >
+                <span className="white">{value}</span>
+            </div>
+        </div>
+    );
+}
+
+function ColorSection({ values }: { values: Color['values'] }): JSX.Element {
+    const [active, setActive] = useState(false);
```

Comment:
It's better to call these `isActive` and `setIsActive` so that we can tell based on the name that it will be a boolean.